tests/e2e/taqueria-cli-scaffolding-functional.spec.ts

```diff
@@ -3,12 +3,16 @@ import fsPromises from 'fs/promises';
 import util from 'util';
 const exec = util.promisify(exec1);
 
-const scaffoldDirName = `taqueria-taco-shop-functional`;
+const scaffoldDirName = `e2e/auto-test-taco-shop-functional`;
 
 describe('E2E Testing for taqueria scaffolding initialization,', () => {
 	beforeAll(async () => {
-		await exec(`taq scaffold https://github.com/ecadlabs/taqueria-scaffold-taco-shop.git ${scaffoldDirName}`);
-		await exec(`cd ${scaffoldDirName} && npm run setup`);
+		await fsPromises.rm(`${scaffoldDirName}`, { recursive: true, force: true });
+
+		// Please leave this console log intact. Its useful to see _when_ there is a failure.
+		console.log(
```

Comment:
We should probably remove this console.log

Reply:
Why? I personally like seeing the output here, as it makes it much easier to debug this in the future should something fail.

Reply:
I think we can add it if there's a failure. It's noisy to console everything even when things are passing.

Reply:
I think we should add a logging mechanism for tests that logs to a file, and saved as an artifact in the job. Otherwise, so much time is wasted in re-adding your console.log statements and waiting for tests to run. I think till that mechanism is in place, I'd strongly advocate for leaving the console.log intact.